ArcGISRuntimeSDKQt_CppSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.qml

```diff
@@ -98,7 +98,7 @@ DistanceMeasurementAnalysisSample {
             ComboBox {
                 id: comboBox
                 property int modelWidth: 0
-                width: modelWidth + leftPadding + rightPadding + indicator.width
+                width: modelWidth + leftPadding + rightPadding
```

Comment:
This doesn't affect the sample functionality and is not reproducible on Windows since we automatically expand the dropdown to accommodate all characters. Not sure why that isn't being honored on Mac but I'd prefer opening a follow up issue for this rather than holding up this PR. Currently as is the dropdown doesn't appear at all because that value is null. 

I can manually add a few extra pixels but I'd rather not hardcode stuff like that. Do you have a better suggestion?

![distance](https://user-images.githubusercontent.com/3746552/192401072-a3626146-04ca-4174-b016-57dadf3015d0.PNG)

Reply:
I agree, we can log an issue to follow up on this